### PR TITLE
Right clicks

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by dannyparsons** [comment]
Don't forget to edit the column metadata right click menu as well :)

**Comment by shadrackkibet** [comment]
@africanmathsinitiative/developers this is ready for review.

**Comment by shadrackkibet** [comment]
@dannyparsons  this is ready for you to review.

I am not sure how to disable labels/levels on rightclicks when the the user selects more than one column and also when the column variable Type isn't a Factor.

File: instat/ucrDataView.vb [hunk 1]
```diff
@@ -460,8 +460,12 @@ Public Class ucrDataView
         clsConvertTo.AddParameter("to_type", Chr(34) & "numeric" & Chr(34))
         clsConvertTo.AddParameter("col_names", SelectedColumns())
         RunScriptFromDataView(clsConvertTo.ToScript(), strComment:="Right click menu: Convert Column(s) To Numeric")
-    End Sub
-
+    End Sub
+
+    Private Sub mnuLevelsLabels_Click(sender As Object, e As EventArgs) Handles mnuLevelsLabels.Click
+        dlgLabelsLevels.ShowDialog()
+    End Sub
```
Comment (dannyparsons) [inline instat/ucrDataView.vb]: Can you make this work like other right click items where the selected column is filled in when I go to the dialog? I think Sort is an example of this.

Comment (shadrackkibet) [reply instat/ucrDataView.vb]: @dannyparsons  this is now done. This is ready for you again.

**Pull request merged**